app/models/provider_enrichment.rb

```diff
@@ -23,7 +23,9 @@ class ProviderEnrichment < ApplicationRecord
 
   belongs_to :provider,
              inverse_of: 'enrichments'
-  audited associated_with: :provider
+
+  audited except: :accrediting_provider_enrichments,
+          associated_with: :provider
```

Comment:
Why do we need to exclude `accrediting_provider_enrichments`?

Reply:
It gets set twice. Once via the JSON Data and then as an attribute.

https://trello.com/c/BVL0yPPD/1916-s-update-provider-audit-trail-when-changing-provider-enrichment#comment-5d5286285da65f384429af02

Reply:
Is this true only for this `jsonb_accessor` attribute? Don't they all do this?

Reply:
Ahh. It does like it's any `jsonb_accessor`

Reply:
So is it worth excluding `jsonb_accessor` then?

Reply:
Yes, looking at that now.